internal/pkg/metadb/metadb.go

```diff
@@ -709,26 +709,26 @@ func addPropertyFilter(q *ds.Query, f *pb.QueryFilter) (*ds.Query, error) {
 
 // QueryRecords returns a list of records that match the given filters and their stores.
 // TODO(https://github.com/googleforgames/open-saves/issues/339): consider refactoring this to fewer arguments.
-func (m *MetaDB) QueryRecords(ctx context.Context, filters []*pb.QueryFilter, storeKey, owner string, tags []string, orders []*pb.SortOrder) ([]*record.Record, []string, error) {
+func (m *MetaDB) QueryRecords(ctx context.Context, req *pb.QueryRecordsRequest) ([]*record.Record, []string, error) {
 	query := m.newQuery(recordKind)
-	if storeKey != "" {
-		dsKey := m.createStoreKey(storeKey)
+	if req.GetStoreKey() != "" {
```

Comment:
I think using the field values is more readable than calling Get*** everywhere but what do you think?

https://github.com/googleforgames/open-saves/issues/319

Reply:
After some offline discussion, it does look like these getters are preferred.